README.md

```diff
@@ -4,42 +4,59 @@ Provides Azure Authentication Middleware for a Laravel App.  If you like this, c
 
 ## Normal Installation
 
-1. `composer require rootinc/laravel-azure-middleware`
-2. run `php artisan vendor:publish --provider="RootInc\LaravelAzureMiddleware\AzureServiceProvider"` to install config file to `config/azure.php`
-3. In our routes folder (most likely `web.php`), add
+1. From the command line, run `composer require rootinc/laravel-azure-middleware`
+2. After Composer has installed the package, run `php artisan vendor:publish --provider="RootInc\LaravelAzureMiddleware\AzureServiceProvider"` to install the config file to `config/azure.php`
+3. Add the login and login callback routes to you router (by default `routes/web.php`) like shown below:
 ```php
-Route::get('/login/azure', '\RootInc\LaravelAzureMiddleware\Azure@azure');
-Route::get('/login/azurecallback', '\RootInc\LaravelAzureMiddleware\Azure@azurecallback');
+Route::get('/azure/login', [\RootInc\LaravelAzureMiddleware\Azure::class, 'azure'])->name('azure.login');
```

Comment:
I totally understand `azure` being first here.  However, we use this package, alongside other authentication methods (for example SAML2, which yields, `login/saml2`).  What are your thoughts on the current convention, where we want to login, but the route after 'login' is the method?  I understand this is just documentation and the developer can implement either way.  Just curious to understand another perspective.
@jaredrhodes

Reply:
I can totally see how having login first has a valid use case. My reasoning is this code change is that showing (as an example) it like this is easier to understand for a new adopter. The `/azure` part logically separates the app into its own namespace as an example so you know that the different routes, `login`, `callback` and eventually `logout` all belong to the same controller.
Having `/login` first, at least for me, makes it less logical to see the connection to the same controller although they are named similarly.

What do you think?

Reply:
I am okay with it as an example.  Since it is an example, those who have adopted the other way won't be hurt by this change.  And I totally see the case for making it easier for new adopters to understand.